src/main/java/frc/robot/subsystems/Elevator.kt

```diff
@@ -143,6 +95,41 @@ object Elevator : SubsystemBase("Elevator") {
             closedLoopError.setUpdateFrequency(100.0)
         }
 
+    val position
+        get() = leader.position.value.toElevatorHeight()
+
+    fun goTo(state: RobotState): Command =
+        PrintCommand("Elevator going to $state - ${state.elevatorHeight}")
+            .alongWith(
+                runOnce {
+                    leader.setControl(
+                        motionMagic.withPosition(state.elevatorHeight.toDrumRotations())
+                    )
+                }
+            )
+
+    val isStowed: Boolean
+        get() = position < IS_STOWED_THRESHOLD
+
+    fun goToAndWaitUntilStowed(state: RobotState): Command =
+        PrintCommand("Elevator stowing").alongWith(goTo(state)).andThen(Commands.idle()).until {
+            isStowed || state.elevatorHeight > IS_STOWED_THRESHOLD
+        }
+
+    val manualUp by command {
+        startEnd(
+            { leader.setControl(voltage.withOutput(2.0.volts)) },
+            { leader.setControl(motionMagic.withPosition(leader.position.value)) },
+        )
+    }
+
+    val manualDown by command {
+        startEnd(
+            { leader.setControl(voltage.withOutput((-2.0).volts)) },
+            { leader.setControl(motionMagic.withPosition(leader.position.value)) },
+        )
+    }
```

Comment:
In the future we can replace this with 
```
val manualUp by command {
  run { 
    leader.setControl(motionMagic.withPosition(leader.position.value + MANUAL_SPEED*dt)
 }
}
```
which will modify the motion magic each position by each value. 

We don't have to change it now